index.html

```diff
@@ -3,15 +3,18 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>A24-list-app</title>
+    <title>Pokemon-list-app</title>
     <link rel="stylesheet" type="text/css" href="css/stylesheet.css">
     <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
   </head>
-  <body>
+  <body id="body">
     <header class="page-header">
-    All A24-Movies in one App
+    All pokemons in one App
   </header>
-    <ul class="movie-list"></ul>
+    <div class="load"></div>
```

Comment:
This div with className `load` should be wrapped around the `ul` otherwise the loading will be infinite.